src/lib/storage/fake_device_factory.rb

```diff
@@ -242,7 +242,7 @@ def create_partition_table(parent, args)
       # @return [Fixnum]
       def str_to_ptable_type(string)
         # Allow different spelling
-        string = "msdos" if string.downcase == "ms-dos"
+        string = "msdos" if string.casecmp("ms-dos").zero?
```

Comment:
I find the old version much better readable. Now you have to lookup casecmp to know that it returns for equal strings, e.g. could be true or 0.